src/app/pages/home-page/home-page.component.html

```diff
@@ -24,4 +24,10 @@
   [image]="'./assets/home-3.png'">
 </app-text-image-component>
 
+<app-idea-form>
+
+</app-idea-form>
```

Comment:
reformat the code a bit: it is better for them to be on the same line: <app-idea-form></app-idea-form>